packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -12,13 +12,13 @@
 
 #### salesforcedx-vscode-core
 
-- Org browser retrieve & open handling types with xml only files ([PR #2635](https://github.com/forcedotcom/salesforcedx-vscode/pull/2635))
+- Org Browser retrieve & open handling types with xml only files ([PR #2635](https://github.com/forcedotcom/salesforcedx-vscode/pull/2635))
```

Comment:
@ekapner I'm addressing your previous changelog feedback on this PR. Let me know what you think

Reply:
thank you, looks good.